system/ioc/Builder.cfc

```diff
@@ -644,7 +644,11 @@ component serializable="false" accessors="true" {
 	 * @definition The dependency definition structure: name, dsl as keys
 	 * @targetObject The target object we are building the DSL dependency for
 	 */
-	private any function getWireBoxDSL( required definition, targetObject ){
+	private any function getWireBoxDSL(
```

Comment:
I think this change needs to be consistent with every DSL get method, not only this one. Also, maybe this needs to update the `coldbox.system.ioc.dsl.IDSLBuilder` the `process()` method so we can standardize it.

Reply:
Also, can you expand on the ticket and here WHY do this?

Reply:
Sure - in Quick I have a way to get a new blank copy of the same entity.  I do this by asking WireBox for it.  But to ask WireBox for it I need to know what mapping was used.  Right now Quick does this using a `beforeInstanceAutowire`interceptor, but I thought this was cleaner.